blog/_posts/2020-12-12-multi-cluster-monitoring.md

```diff
@@ -0,0 +1,256 @@
+---
+layout: post
+title: Multi-Cluster monitoring with Prometheus and inlets
+description: In many cases, a Kubernetes environment consist of multiple clusters. Learn how inlets can help setting up Prometheus Federation and bringing all the metrics in a single dashboard.
+author: Johan Siebens
+tags: inlets-pro hybrid-cloud metrics prometheus federation helm
+author_img: jsiebens
+image: /images/2020-12-multi-cluster-monitoring/banner.jpg
+date: 12-12-2020
+---
+
+## Introduction
+
+More and more organisations are spreading their workloads across multiple Kubernetes cluster to improve availability and scalability. Perhaps you are running some batch jobs on Amazon Spot instances to reduce cost, while your heavy processing services are more likely benefit the power of bare-metal instances. From an operation point of view, it can become quite a challenge to monitor all those different clusters, running on various cloud providers or on-premises, all of them in an isolated network.
```

Comment:
The lead sentence has no source to back up the quote: "More and more organisations are spreading their workloads across multiple Kubernetes cluster"

How about?

There are two main reasons why you may have more than one Kubernetes cluster to monitor. The first use-case is where you own the clusters and have one per stage of development such as development, staging and production. Another is where you run a managed service, or have clients, who run workloads that you need to monitor for reliability, or for consumption as part of a service you run.

I.e. 

1 - internal use case. Different dev stages. Or different team/project clusters. For instance, a Monzo or a Form3.
2 - you are a managed service provider like a NetApp or a VSHN (on the homepage).